skysat/ndvi/script.js

```diff
@@ -1,22 +1,22 @@
 //VERSION=3
 function setup() {
   return {
-    input: [{"bands": ["Red", "NIR","dataMask"]}],
-    output: { bands: 4}
+    input: [{ "bands": ["Red", "NIR", "dataMask"] }],
```

Comment:
I would simplify the input:

```
input: ["Red", "NIR", "dataMask"],
```

Unless you have a good reason to keep the nomenclature like you did.